1.wave2_data/0.download_data/README.md

```diff
@@ -0,0 +1,21 @@
+# Download data for wave 2: ASC Bleedthrough Assessment
+
+In this module, we will provide the instructions for downloading the wave 2 data, which is a modified Cell Painting assay consisting of 4 channels totalling 4,608 images images. There are 2 conditions within this wave:
+
+For **condition 7**:
+- HOECHST 33342 -> Nuclei
+- Alexa 488 -> ASC
+- Mito_641_narrow -> Mitochondria
+- Alexa 700_narrow -> absolute spill over form mito 641
+
+For **condition 8**:
+- HOECHST 33342 -> Nuclei
+- Alexa 488 -> nothing/autofluorescence 
+- Mito_641_narrow -> Mitochondria
+- Alexa 700_narrow -> ASC
```

Comment:
Was this information sourced from elsewhere? If so, consider citing where it is located (using a link if possible).

Reply:
It is sourced from a document on the shared Google Drive, but the document does contain "confidential - do not copy".  If I could, I would make the document accessible to the public and link it but I don't want to overstep.

For a future PR @gwaybio, are we able to make that document public and link it? Or does it contain a lot of info that the company would not want out there (expect for the plate and conditions).